### PR TITLE
[Backport 1.3] Backport Demo Config Script -y Function

### PR DESCRIPTION
**Pull request by scrawfor99**
### Description
Backports PR #2308 to 1.3 

The -y flag in the demo scripts was not actually setting the config settings correctly.

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]

File: .github/workflows/plugin_install.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        jdk: [8, 11, 14]
```
Comment (peternied) [inline .github/workflows/plugin_install.yml:15]: This shouldn't be updated

Comment (scrawfor99) [reply .github/workflows/plugin_install.yml:15]: Yeah my bad, I just copy+pasted and did not catch that thank you!

**Comment by scrawfor99** [comment]
There is no sanity tests and a different file naming convention with 1.3 since the assemble creates two instances of the security plugin and there is no integration test file.

**Comment by scrawfor99** [comment]
@peternied @cwperks I think that this may just be a flaky test. Could one of you rerun it for me?

**Comment by cwperks** [comment]
I just started re-running the failed job.

**Review by peternied — APPROVED** [review]

**Pull request merged**